jetbrains-core/src/software/aws/toolkits/jetbrains/core/executables/ExecutableManager.kt

```diff
@@ -51,12 +55,13 @@ class DefaultExecutableManager : PersistentStateComponent<List<ExecutableState>>
         }
     }
 
-    override fun getExecutableIfPresent(type: ExecutableType<*>): ExecutableInstance = internalState[type.id]?.second?.takeIf {
-        when (it) {
-            is ExecutableWithPath -> it.executablePath.exists()
-            else -> true
-        }
-    } ?: ExecutableInstance.UnresolvedExecutable()
+    override fun getExecutableIfPresent(type: ExecutableType<*>): ExecutableInstance =
```

Comment:
This looks like formatting too - what's your line length set to? Should be 160

Reply:
We need to add that to the style guide, default kotlin settings set it to 120

Reply:
Fixed the formatting